src/jepsen/voltdb.clj

```diff
@@ -234,8 +234,9 @@
   "Takes an SQL query and runs it on the local node via sqlcmd"
   [query]
   (c/cd base-dir
-        (c/sudo username
-                (c/exec "bin/sqlcmd" (str "--query=" query)))))
+      (c/exec (c/env {"JAVA_HOME" (System/getenv "JAVA_HOME")
+                      "PATH" (System/getenv "PATH")})
```

Comment:
either remove PATH or add it everywhere.
Probably a comment here would be helpful.